backend/databasemodel/model.sql

```diff
@@ -1510,17 +1530,17 @@ ALTER TABLE kooste.museovirastoarcrest_rkykohteet OWNER TO tarmo_admin;
 CREATE TABLE kooste.museovirastoarcrest_muinaisjaannokset (
 	mjtunnus bigint NOT NULL,
 	geom geometry(MULTIPOINT, 3067) NOT NULL,
-	nakyvyys boolean NOT NULL DEFAULT True,
-	kohdenimi text,
-	laji text,
```

Comment:
Here we should probably also keep all the original fields that are not saved to `name`? Also, what is `cityName`, couldn't find that field anywhere else?

Reply:
Looks like that the idea was not to remove fields, because upgrade.sql does not remove fields?

Reply:
A little bit same here as with lisatietoja field in luontorastit table. I changed my mind about the relevance of that laji field. Main reason was that we already have "type information" available via tyyppi field and tarmo category so I am not sure if there is a need to show the user info whether object is kiinteä muinaisjäännös or muu kulttuuriperintökohde.

What did you mean by upgrade not removing the fields? I guess laji field is removed in row 246 in upgrade.sql?

About kunta field, I just thought that this field name would be nice to unify with any other field (in other tables) storing same kind of information. Since this kind of information exists also in lipas tables, I renamed kunta field according to its name. Can remove this change too, if you wish. It was just mean to clarify connections :)

Reply:
Okay, so how do we know we need all the fields apart from `laji` from the list below? There are lots of fields in *all* tables here that we might not need in the UI. I think it's better that we keep all the data here at the moment?

<img width="709" alt="image" src="https://user-images.githubusercontent.com/9367712/160393302-9bcbbe0e-ea9f-44a1-9cb9-b9d3eebcffd6.png">

Another option, of course, is go through all the data fields in all the data sources properly, and remove extra fields. This way, we could e.g. combine the museovirasto layers to the same table, if we want to get rid of the extra fields that we don't use. Of course, this would make it kinda useless to have lots of different table schemas, if we save only the common fields after all.

I just don't think we should be removing random data in some layers one field at a time, at this point. We have dozens of fields in all the tables that we don't need right now. But we should go through all of them then, and might make wrong guesses and have to add them back again later. Better store all the incoming data, I guess that was the idea behind having different tables for each layer?

"cityName" makes sense if that is where we want to store the name of the city in each table. That makes it a common field for all tables. Of course, the right way to do it might be to use the location data of the geometry, so that we are sure the city is always correct?

Do you know if there are other fields in some other tables that are common to lipas fields? Should we check them too in a separate issue?

Reply:
You are probably correct.

The original selection of the gathered field data (note that we do not currently store all of the data you listed above in database table; e.g. inspireID, paikannustapa, paikannustarkkuus, selite, x, y, shape are not imported) was based just on what I assumed might become handy now or later on. So I thought it would be ok to keep on removing these fields as randomly if it starts to feel they are redundant. In other words I looked at the data in Tampere region via QGIS and noticed that laji field only contained two possible values (kiinteä muinaisjäännös, muu kulttuuriperintökohde) and thought there is no point in storing this data. I acknowledge this is random and incremental way to find the suitable combination of attribute data but I at the same time I regret that I chose to include this field in the first phase..

Reply:
Regarding this question, Do you know if there are other fields in some other tables that are common to lipas fields? Should we check them too in a separate issue?

I have checked all field names and contents in this PR and unified the field names when applicable. So there should not be any need for creating a separate issue on this.

Reply:
Ah okay, then it is alright if we are sure the data will not be needed any more! Admittedly I didn't check all the fields and their contents.